File: Javascript/MinimumDifferenceBetweenHighestAndLowestOfKScores.js
//Leetcode Problem - 1984

var minimumDifference = function(nums,k){
    let minValue = 0
    nums.sort((a,b) => a-b)

    for (let i =k-1;i<nums.length;i++){
        if (i === (k-1)){
            minValue = nums[i] - nums[i-(k-1)]
            continue
        }
        minValue = Math.min(minValue,nums[i] - nums[i-(k-1)])

    }

    return minValue
}

File: Python/MinimumDifferenceBetweenHighestAndLowestOfKScores.py
##Leetcode Problem - 1984

def minimumDifference(nums,k):
    minValue = 0
    nums.sort()

    for i in range(k-1,len(nums)):
        if i == (k-1):
            minValue = nums[i] - nums[i-(k-1)]
            continue

        minValue = min(minValue, nums[i]-nums[i-(k-1)])


    return minValue